UnityProject/Assets/Scripts/Doors/AirLockAnimator.cs

```diff
@@ -195,7 +195,8 @@ public override void CloseDoor(bool skipAnimation)
 		{
 			if (offset > -1 && numberOfSpritesToPlay > 0)
 			{
-				int limit = offset + numberOfSpritesToPlay;
+				// clamp to make sure that index is not out of range
+				int limit = Mathf.Clamp(offset + numberOfSpritesToPlay, 0, list.Length-1);
 				if (skipToEnd)
```

Comment:
IndexOutOfRangeException: Index was outside the bounds of the array.
AirLockAnimator+<PlayAnim>d__13.MoveNext () (at /github/workspace/UnityProject/Assets/Scripts/Doors/AirLockAnimator.cs:207)
UnityEngine.SetupCoroutine.InvokeMoveNext (System.Collections.IEnumerator enumerator, System.IntPtr returnValueAddress) (at /home/builduser/buildslave/unity/build/Runtime/Export/Scripting/Coroutines.cs:17)
UnityEngine.MonoBehaviour:StartCoroutine(IEnumerator)
AirLockAnimator:OpenDoor(Boolean) (at /github/workspace/UnityProject/Assets/Scripts/Doors/AirLockAnimator.cs:107)
DoorAnimator:PlayAnimation(DoorUpdateType, Boolean) (at /github/workspace/UnityProject/Assets/Scripts/Doors/DoorAnimator.cs:22)
DoorUpdateMessage:Process() (at /github/workspace/UnityProject/Assets/Scripts/Messages/Server/DoorUpdateMessage.cs:17)
GameMessageBase:Process(NetworkConnection) (at /github/workspace/UnityProject/Assets/Scripts/Messages/GameMessageBase.cs:25)
GameMessageBase:PreProcess(NetworkConnection, GameMessageBase) (at /github/workspace/UnityProject/Assets/Scripts/Messages/GameMessageBase.cs:18)
Mirror.<>c__DisplayClass6_0`2:<MessageHandler>b__0(NetworkConnection, NetworkReader, Int32) (at /github/workspace/UnityProject/Assets/Mirror/Runtime/MessagePacker.cs:146)
Mirror.NetworkConnection:InvokeHandler(Int32, NetworkReader, Int32) (at /github/workspace/UnityProject/Assets/Mirror/Runtime/NetworkConnection.cs:219)
Mirror.NetworkConnection:TransportReceive(ArraySegment`1, Int32) (at /github/workspace/UnityProject/Assets/Mirror/Runtime/NetworkConnection.cs:272)
Mirror.ULocalConnectionToServer:Update() (at /github/workspace/UnityProject/Assets/Mirror/Runtime/LocalConnections.cs:110)
Mirror.NetworkClient:Update() (at /github/workspace/UnityProject/Assets/Mirror/Runtime/NetworkClient.cs:259)
Mirror.NetworkManager:LateUpdate() (at /github/workspace/UnityProject/Assets/Mirror/Runtime/NetworkManager.cs:255)